benches/simulation_bench.rs

```diff
@@ -74,50 +74,53 @@ fn take_simulation_step(c: &mut Criterion, id: &str, request: SimulationStepRequ
 fn simulation(c: &mut Criterion) {
     let mut loader = bench_helper::get_uni_loader();
 
-    let start_request_1 =
-        construct_start_request(&[loader.get_component("Machine").clone()], "(Machine)", 1);
+    let start_request_1 = construct_start_request(
+        &[loader.get_component("Machine").unwrap().clone()],
+        "(Machine)",
+        1,
+    );
 
     let start_request_2 = construct_start_request(
         &[
-            loader.get_component("HalfAdm1").clone(),
-            loader.get_component("HalfAdm2").clone(),
+            loader.get_component("HalfAdm1").unwrap().clone(),
```

Comment:
I dont like that you use unwrap everywhere. It would be much better if we could handle the error in some way. Atleast editing this to expect would be better. Unless unwrap produces a good error?

Reply:
The errors thrown by get_component unwraps are syntax errors, which are unrecoverable anyway.